### PR TITLE
Update pricing page

### PR DESCRIPTION
**Pull request by broskoski**
pricing-page
screen shot 2019-02-19 at 8 18 45 pm

Still to do:
- [x] Figure out "Create Group" link

### CODE REVIEW

File: react/pages/about/PricingPage/components/PricingTable/index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  flex-direction: row;
+  justify-content: center;
+  
+  ${constants.media.mobile`
+    align-items: center;
+    flex-direction: column;
+  `}
+`;
+
+const Cell = styled.div`
```
Comment (dzucconi) [inline react/pages/about/PricingPage/components/PricingTable/index.js]: Use Box so you get the space mixin

File: react/pages/about/PricingPage/components/PricingTable/index.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  display: flex;
+  flex-direction: column;
+  width: 20em;
+  padding: 1em;
```
Comment (dzucconi) [inline react/pages/about/PricingPage/components/PricingTable/index.js]: use said space mixin

File: react/pages/about/PricingPage/components/PricingTable/index.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  border-right: 2px solid ${x => x.theme.colors.gray.light};
+  
+  &:last-child {
+    border-right-color: transparent;
+  }
+
+  ${constants.media.mobile`
+    width: 100%;
+    border-right: none;
+    padding: 3em 2em;
```
Comment (dzucconi) [inline react/pages/about/PricingPage/components/PricingTable/index.js]: etc.

File: react/pages/about/PricingPage/components/PricingTable/index.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  flex-direction: column;
+  width: 20em;
+  padding: 1em;
+  border-right: 2px solid ${x => x.theme.colors.gray.light};
+  
+  &:last-child {
+    border-right-color: transparent;
+  }
+
+  ${constants.media.mobile`
```
Comment (dzucconi) [inline react/pages/about/PricingPage/components/PricingTable/index.js]: I find the responsive array syntax to be a lot nicer

File: react/pages/about/PricingPage/components/PricingTable/index.js [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+const PlanSubPrice = styled(Text).attrs({
+  f: 4,
+  color: 'gray.base',
+})``;
+
+const CTAButton = styled(Button).attrs({
+  f: 5,
+  flex: 1,
+  mt: 9,
+})``;
```
Comment (dzucconi) [inline react/pages/about/PricingPage/components/PricingTable/index.js]: We have a filled button?

File: react/pages/about/PricingPage/index.js [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+import constants from 'react/styles/constants';
+import Box from 'react/components/UI/Box';
+import Text from 'react/components/UI/Text';
+
+import { Headline, Subheadline, Description } from 'react/pages/about/components/Text';
+import CenterBox from 'react/pages/about/components/CenterBox';
+import PricingTable from 'react/pages/about/PricingPage/components/PricingTable';
+
+const Container = styled.div`
+  margin-top: 4em;
```
Comment (dzucconi) [inline react/pages/about/PricingPage/index.js]: There's rarely an instance where you need to be inputting values manually, if so it's usually for a very good reason.

**Comment by broskoski** [comment]
@dzucconi how now?

**Pull request merged**